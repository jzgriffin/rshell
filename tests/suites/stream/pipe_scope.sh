(echo abc) | rev
(echo def; echo ghi) | rev
(echo jkl && echo mno) | tr a-z A-Z
echo pqr | (rev; echo stu)
