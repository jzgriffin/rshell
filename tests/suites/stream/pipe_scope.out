cba
fed
ihg
JKL
MNO
rqp
stu
